google-cloud-translate/README.md

```diff
@@ -126,13 +126,13 @@ import com.google.cloud.translate.Translate.TranslateOption;
 import com.google.cloud.translate.Translation;
 ```
 
-Then add the following code to translate a text (specifying its source language):
+Then, for example to translate from English to Spanish, add the following code:
 
 ```java
 Translation translation = translate.translate(
     "World",
     TranslateOption.sourceLanguage("en"),
-    TranslateOption.targetLanguage(detectedLanguage));
```

Comment:
In the larger context of the README, it seems like we should use `detectedLanguage` (for source language). I took a peek at https://github.com/GoogleCloudPlatform/google-cloud-java/blob/70d98a29e36474a73be93781b49f4e3d54bef8ad/google-cloud-examples/src/main/java/com/google/cloud/examples/translate/snippets/DetectLanguageAndTranslate.java , and it looks like its code doesn't match the README code (which the README claims it does). We should fix that too in this PR.